drivers/amazonec2/amazonec2.go

```diff
@@ -335,10 +349,35 @@ func (d *Driver) Create() error {
 	}
 
 	log.Debugf("launching instance in subnet %s", d.SubnetId)
-	instance, err := d.getClient().RunInstance(d.AMI, d.InstanceType, d.Zone, 1, 1, d.SecurityGroupId, d.KeyName, d.SubnetId, bdm, d.IamInstanceProfile)
-
-	if err != nil {
-		return fmt.Errorf("Error launching instance: %s", err)
+	var instance amz.EC2Instance
+	if d.RequestSpotInstance {
+		spotInstanceRequestId, err := d.getClient().RequestSpotInstances(d.AMI, d.InstanceType, d.Zone, 1, d.SecurityGroupId, d.KeyName, d.SubnetId, bdm, d.IamInstanceProfile, d.SpotPrice)
+		if err != nil {
+			return fmt.Errorf("Error request spot instance: %s", err)
+		}
+		var instanceId string
+		var spotInstanceRequestStatus string
+		log.Infof("Waiting for spot instance fullfillment...")
+		log.Infof("This will take about 3 minutes.")
```

Comment:
This should be removed.  We probably don't want to give timing estimates because they can vary widely.